plugin/backend.go

```diff
@@ -120,6 +121,16 @@ func (pm *Manager) Push(name string, metaHeader http.Header, authConfig *types.A
 	}
 	defer config.Close()
 
+	var dummy map[string]interface{}
```

Comment:
Why not unmarshal to `types.Plugin`? We could even validate the type for some required keys.